packages/trader/src/Modules/CFD/Components/cfd-poa.tsx

```diff
@@ -0,0 +1,544 @@
+                                            )}
+                                            {poa_status === PoaStatusCodes.pending && (
+                                                <PoaNeedsReview is_description_enabled={false} />
+                                            )}
+                                            {isVerified && (
+                                                <PoaVerified is_description_enabled={false} has_poi={has_poi} />
+                                            )}
+                                            {poa_status === PoaStatusCodes.expired && (
+                                                <PoaExpired onClick={handleResubmit} />
+                                            )}
+                                            {(poa_status === PoaStatusCodes.rejected ||
+                                                poa_status === PoaStatusCodes.suspected) && <PoaUnverified />}
+                                        </ThemedScrollbars>
+                                    )}
+                                    <Modal.Footer is_bypassed={isMobile()}>
+                                        {(isVerified || is_form_visible) && (
+                                            <FormSubmitButton
+                                                has_cancel
+                                                cancel_label={localize('Previous')}
+                                                is_disabled={
+                                                    isFormDisabled(dirty, errors) ||
+                                                    (!isVerified &&
+                                                        document_upload.files &&
+                                                        document_upload.files.length < 1) ||
+                                                    !!document_upload.error_message
+                                                }
+                                                label={isVerified ? localize('Submit') : localize('Next')}
+                                                is_absolute={isMobile()}
+                                                is_loading={isSubmitting}
+                                                form_error={form_error}
+                                                onCancel={() => handleCancel(values as TFormValues)}
+                                            />
+                                        )}
+                                    </Modal.Footer>
+                                </Div100vhContainer>
+                            </form>
+                        )}
+                        ;
+                    </AutoHeightWrapper>
+                );
+            }}
+        </Formik>
+    );
+};
+
+CFDPOA.propTypes = {
+    onCancel: PropTypes.func,
+    onSave: PropTypes.func,
+    onSubmit: PropTypes.func,
+    storeProofOfAddress: PropTypes.func,
+    value: PropTypes.object,
+};
```

Comment:
One main point of migration to TS is to remove `propTypes` and strictly type our react code, We don't need to keep `propTypes` here anymore!

Reply:
Removed